### ISSUE TITLE
Cinnamon Menu Consistency Issue

### ISSUE THREAD

**Issue by HyperHamster** [numixproject/numix-icon-theme-circle#2164]
![numixgitissue](https://cloud.githubusercontent.com/assets/9900991/7785988/90b9487a-0174-11e5-8dd6-8c91fdcce577.png)

Issue is illustrated in above image.

I would give an example symlink but there doesn't seem to be a Numix Circle equivalent for that controller icon, I do however know that the name of that icon is "applications-games". This also doesn't seem to be the only "applications-" prefix icon that is missing an icon or symlink either, fixing this would probably prevent further consistency issues.

**Comment by bil-elmoussaoui** [comment]
Well, i think this is due to the categories icons. The Numix designers didn't decide yet if they will make the categories icons circular or not. As those icons are used in Software center! and personally i do prefer the current categories icons instead of the circular ones

**Comment by wa4557** [comment]
I think it should be the other way around: Remove those icons from circle so that the categories icons from base are used, instead of the circular icons

**Comment by bil-elmoussaoui** [comment]
@wa4557 i'm tottally agree with you. The same goes for the ubuntu online accounts , i have no idea why those icons are in Numix circle even if they're not.

**Comment by wa4557** [comment]
@HyperHamster Can you post a screenshot of what happens if you use Numix instead of Numix-Circle as icon theme?

**Comment by bil-elmoussaoui** [comment]
This is a screenshot using Cinnamon with Numix base
![screenshot from 2015-05-25 09 11 00](https://cloud.githubusercontent.com/assets/7660997/7794872/9b94cefe-02be-11e5-8cb9-3e23001160b9.png)

**Comment by wa4557** [comment]
Ok so it seems the corresponding icons that are in base and do not belong in circle are:
```
applications-accessories
applications-graphics
applications-internet
applications-office
applications-multimedia
preferences-desktop (probably)
```
is there a reason why they are in circle, is it possible to remove those? @dirtydancing I wasn't able to find a place in Ubuntu where they are used, do you have any idea why they are in circle?

**Comment by dirtydancing** [comment]
No idea why these icons are in circle :) I have also not come across these in Ubuntu. I have recently moved from Ubuntu/Unity to Arch/GNOME Shell, and so far these icons have not come up there either.

**Comment by wa4557** [comment]
Cool, so you're now a first hand GNOME-shell tester perfect :)

@bil-elmoussaoui @dirtydancing Can you run somehting like `grep -r "Icon=applications-" .` in `/usr/share/applications/` this should list all the files that use those category icons as app icon. 

In Unity only `applications-system` is used for `unity-info-panel`, but since those icons are the same in circle and base there's no conflict here, and the icon could be removed from circle.

**Comment by bil-elmoussaoui** [comment]
The result in gnome shell (Gnome 3.16) : 
```
/usr/share/applications/gnome-info-panel.desktop:Icon=applications-system
/usr/share/applications/nautilus-connect-server.desktop:Icon=applications-internet
```

**Comment by bil-elmoussaoui** [comment]
In Cinnamon the result of your command is nothing, so i tried this instead
```
 grep -r "Icon=cs-" /usr/share/applications/*

```
I dont think that they re the same icons but this is somehow related to #1930 
```
usr/share/applications/cinnamon-color-panel.desktop:Icon=cs-color
/usr/share/applications/cinnamon-datetime-panel.desktop:Icon=cs-date-time
/usr/share/applications/cinnamon-display-panel.desktop:Icon=cs-display
/usr/share/applications/cinnamon-network-panel.desktop:Icon=cs-network
/usr/share/applications/cinnamon-region-panel.desktop:Icon=cs-language
/usr/share/applications/cinnamon-settings-applets.desktop:Icon=cs-applets
/usr/share/applications/cinnamon-settings-backgrounds.desktop:Icon=cs-backgrounds
/usr/share/applications/cinnamon-settings-calendar.desktop:Icon=cs-date-time
/usr/share/applications/cinnamon-settings-default.desktop:Icon=cs-default-applications
/usr/share/applications/cinnamon-settings-desklets.desktop:Icon=cs-desklets
/usr/share/applications/cinnamon-settings-desktop.desktop:Icon=cs-desktop
/usr/share/applications/cinnamon-settings-effects.desktop:Icon=cs-desktop-effects
/usr/share/applications/cinnamon-settings-extensions.desktop:Icon=cs-extensions
/usr/share/applications/cinnamon-settings-fonts.desktop:Icon=cs-fonts
/usr/share/applications/cinnamon-settings-general.desktop:Icon=cs-general
/usr/share/applications/cinnamon-settings-hotcorner.desktop:Icon=cs-overview
/usr/share/applications/cinnamon-settings-info.desktop:Icon=cs-details
/usr/share/applications/cinnamon-settings-keyboard.desktop:Icon=cs-keyboard
/usr/share/applications/cinnamon-settings-mouse.desktop:Icon=cs-mouse
/usr/share/applications/cinnamon-settings-notifications.desktop:Icon=cs-notifications
/usr/share/applications/cinnamon-settings-panel.desktop:Icon=cs-panel
/usr/share/applications/cinnamon-settings-power.desktop:Icon=cs-power
/usr/share/applications/cinnamon-settings-privacy.desktop:Icon=cs-privacy
/usr/share/applications/cinnamon-settings-screensaver.desktop:Icon=cs-screensaver
/usr/share/applications/cinnamon-settings-themes.desktop:Icon=cs-themes
/usr/share/applications/cinnamon-settings-tiling.desktop:Icon=cs-tiling
/usr/share/applications/cinnamon-settings-user.desktop:Icon=cs-user
/usr/share/applications/cinnamon-settings-windows.desktop:Icon=cs-windows
/usr/share/applications/cinnamon-settings-workspaces.desktop:Icon=cs-workspaces
/usr/share/applications/cinnamon-sound-nua-panel.desktop:Icon=cs-sound
/usr/share/applications/cinnamon-sound-panel.desktop:Icon=cs-sound
/usr/share/applications/cinnamon-universal-access-panel.desktop:Icon=cs-universal-access
/usr/share/applications/cinnamon-wacom-panel.desktop:Icon=cs-tablet

**Comment by dirtydancing** [comment]
@wa4557 Yeah, now that it looks like Ubuntu/Unity got Numix-completed, it was time to move on :)

I get the same results as @bil-elmoussaoui with Gnome 3.16:
```
gnome-info-panel.desktop:Icon=applications-system
nautilus-connect-server.desktop:Icon=applications-internet
```

In Gnome 3.16, `applications-system.svg` is used in the system settings overview (for "Details" in German).

This is why I would not recommend to remove this circle icon, because all the other icons in the system settings overview are also circle-icons. It is arguably not recommendable to rely on the circle shape of a base icon, as the base icons are subject to change, which includes change of shape.

Besides, how about Numix-Square, which inherits from Numix base as well? The base icon cannot be circle-shaped and square-shaped at the same time :)

**Comment by bil-elmoussaoui** [comment]
The best solution is to make the categories icons circle/square. The will fix a lot of issues as too many apps uses/settings uses the categories icons

**Comment by wa4557** [comment]
Ok so removing these icons (except `applications-system`) is probably the way to go, since they aren't used anywhere else. Making the `categories` icons circular is not an option imo, because numix-base has nothing to do with circle and if someone uses square that is a problem...

The `Icon=cs-*` are ok, because those icons are indeed circular in numix-circle and are not present in numix-base

**Comment by bil-elmoussaoui** [comment]
Before doing this, some applications are symlinked to applications-* like online-accounts, for me we have to remove those icons 
```
applications-fonts.svg
applications-internet.svg
applications-php.svg
applications-python.svg
applications-science.svg
applications-other.svg
```
@wa4557 there's a way to see if there's any symlinked icon to one of those icons?

**Comment by wa4557** [comment]
not a simple one that I know of. What's the list of icons you want to remove?

**Comment by HyperHamster** [comment]
Whether the Numix developers choose to remove the circular application categories or choose to make circular versions for all of them is irrelevant, it's the inconsistency between the two that is the issue.

But, if I had to chime in to this debate over circular or not, I'd definitely pick circular. If these icons were used only in application launchers such as Cinnamon's menu, Xfce's menu, etcetera, the non-circular versions wouldn't be an issue but they aren't only used there. I don't really know the design manifesto for Numix Circle but it seems to me that circle-less icons are only used at small resolutions, such as context menus, and since these category icons are used at much higher resolutions by numerous applications it disrupts the consistency of the icon pack.

**Comment by bil-elmoussaoui** [comment]
@HyperHamster those icons are used in categories. it's a cinnamon/Xfce issue, i really don't know why some devs keep using the same icon everywhere( the icon name) as they can create a symlink and make the user "free" to chose which icon he wants to use

**Comment by Foggalong** [comment]
My personal opinion on this is that we should keep the non-uniform categories in base, but provide circular versions in Circle and square versions in Square. However we've never reached an agreement officially on how to proceed on this matter.

**Comment by wa4557** [comment]
That's fine as well of course, but then a lot of icons are missing in circle (and square). In order to fix this issue its `applications-games`

**Comment by bil-elmoussaoui** [comment]
@Foggalong if we do so, the icons in categories must be circle too.. as almost all the software centers uses those icons for apps categories

**Comment by Foggalong** [comment]
I thought we were talking about the icons in categories?

**Comment by wa4557** [comment]
@bil-elmoussaoui The icons in base can stay the same, they shouldn't be affected by changes in circle. But to keep a consistent desktop it is necessary to add a few icons in circle. I had a quick look and symlinks should be sufficient...

**Comment by wa4557** [comment]
I think however that it is not reasonable to have these icons also in circle. If you look for example at the above screenshot (the first one) I think it makes more sense to have category icons from base displayed on the lefthand side. After all those are not apps but apps categories therefore it makes more sense to have non circular icons there the same way as for example in the top panel (there aren't circle icons either)...

**Comment by dirtydancing** [comment]
I think I agree with @wa4557 here: The categories icons ("applications-"-named ones) are not apps themselves, so it makes sense to separate them from the Circle apps icons and have them in base. The categories icons do not open an app themselves, but only show more Circle app icons.

This is also a distinction to the system settings "preferences-"-named icons, which are Circle icons: these actually open settings, and do not merely lead to a display of more Circle app icons.

But as far as I understand it, this distinction is blurred by apps that actually use category icons as app icons? And this is why there can be no such distinction, because it leads to inconsistencies? I am not in active contact with categorie-based menus, so this is just an impression from a distance.

**Comment by wa4557** [comment]
@dirtydancing Every app that uses category icons as app icon is buggy, period. That's my opinion

**Comment by dirtydancing** [comment]
Agreed :) Still, the user is left with a noticeable inconsistency, also depending on how widespread this is.

**Comment by dirtydancing** [comment]
It seems that this is not limited to app icons causing problems. Take e.g. this screenshot of the default menu in Xubuntu 15.04:

![xubuntu1504](https://cloud.githubusercontent.com/assets/7164227/7880415/6e61aab8-05f9-11e5-954a-88ed4ea9f132.png)

Just as in the Cinnamon screenshot in the first comment above, only the Games category is showing a non-Circle icon from base. The rest of the category icons are Numix-Circle icons, except for the separated icons for favourites and recent.

And immediately the next problem becomes visible: For "System", `applications-system.svg` is used. This is the very same icon that GNOME is using for "Details" in the system preferences (cf. above). So even if all the other category icons were removed in Numix-Circle, this one would remain and be another inconsistency.

Given this, I believe the best approach to this issue is what @Foggalong prefers as well: keep the present set of "applications-"-named icons in base and add a corresponding set to Circle and accordingly to Square.

I do not know if the set in Circle/Square has to encompass all the current category icons in base. For both Cinnamon and Xfce (as in the Xubuntu default), apparently adding `applications-games.svg` to Circle would do the trick.

Possibly this is a different story for the Ubuntu Software Center. As far as I can tell, currently most of the categories in the Ubuntu Software Center (or more precisely sub-categories for some of the categories) are displayed with base icons. I do not know if these sub-category icons also show up somewhere in Cinnamon/Xfce outside of the USC.

To sum up, arguably the most efficient way to deal with this issue would be to add only `applications-games.svg` to Circle/Square and to see if any other category icon comes up for the relevant launchers.

**Comment by palob** [comment]
This issue was brought up before ( #1580  ) with a bounty on it. Hence a few more cirular category icons were made ( #1678 #1706 ).
To me circular category icons seem OK with the Xfce Whisker Menu but with Cinnamon I'd prefer base ones.

So given the GNOME system settings overview issue the only way to achieve consistency across DEs would be Numix Circle category icons?

**Comment by dirtydancing** [comment]
I am not sure whether a "formal and structured" approach is useful in this case. With so many different DE and different DE versions around, a pragmatic "issue by issue" approach seems to be the way to go.

Here, I see two screenshots of the prominent Cinnamon and Xfce (Whisker) Menu, and in both cases one single icon is inconsistent. So I would recommend to simply include that one icon in Numix-Circle.

Please keep in mind, this is the default offering. If users really want base icons for the categories, they are free to intervene manually and remove these icons from Circle. But the default should be consistent.

**Comment by wa4557** [comment]
I think for now we should just add `application-games` to circle until you guys have figured out what you want. If it turns out that you want non circular categories icons (which I still think is the way to handle this bug) the icon still can be removed
